mediathread/settings_shared.py

```diff
@@ -23,17 +23,15 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'mediathread',
+        'NAME': 'camron',
         'HOST': '',
-        'PORT': '',
-        'USER': '',
-        'PASSWORD': '',
+        'PORT': '5432',
```

Comment:
To customize your local environment, add a local_settings.py in the <app>/mediathread directory. (Alongside the settings_shared.py file). Settings in local_settings.py automatically override the default values in settings_shared.py. local_settings.py is not checked into the repository. Making changes directly to settings_shared.py (as here) can globally affect other installations.